Dependencies/napi/napi-common/Include/napi/napi_pointer.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
```

Comment:
Are you planning on contributing this to N-API? If not, I don't know if it belongs here.

Reply:
I completely forgot about that, to be honest, and made this change without any consideration to where else N-API might be living. My memory/understanding of the issues involved is probably outdated, happy to chat to brush up, but it sounds like it might end up being worthwhile to undo the unification part of this change. I'm also happy with the N-API pointer utility living somewhere else, like its own dependency or a N-API extensions collection or something; just let me know where you think it should go.

Reply:
I don't have a huge preference where it should go, but I don't think it can go here unless we want to contribute it back.